lib/internal/modules/cjs/loader.js

```diff
@@ -188,6 +189,11 @@ for (const { 0: id, 1: mod } of NativeModule.map) {
   }
 }
 
+let allBuiltins = [];
+builtinModules.forEach((builtin) => allBuiltins.push(builtin) && allBuiltins.push(`node:${builtin}`));
```

Comment:
Another way to avoid mutating `builtinModules`:

```suggestion
let allBuiltins = builtinModules.map((builtin) => `node:${builtin}`);
```

Reply:
Ah, good old slice.